community/faas/docs/testing.md

```diff
@@ -0,0 +1,77 @@
+# tests
+
+This project's consistency and quality control are backed by simple integration
+tests using the [Bats testing framework](https://github.com/sstephenson/bats).
+
+
+## Requirements
+
+To install bats, follow the instructions in the Bats website above, or execute
+the *development make target*:
+
+```
+make development
+```
+
+
+## Setting up the testing environment
+
+Since most companies and persons using or developing against this project will
+be working under a different GCP Organization, some of the base settings needed
+for these tests to run were abstracted, and moved to a *personalized*
+configuration file, which by default is expected to be `~/.faas-tests.conf`,
+though the location of this tests config file can be customized by setting the
+variable `FAAS_CONF`. For example:
+
+```
+export FAAS_CONF=/etc/faas-tests.conf`
```

Comment:
Could we use environment variables instead of a .conf file? They're easier for CI systems to input.

Reply:
Yes. Adding that.